packages/console/src/SerialConsole/SerialConsole.js

```diff
@@ -110,7 +112,7 @@ class SerialConsole extends React.Component {
     }
 
     return (
-      <div className={topClassName} id={id}>
+      <div className={`serial-console-pf ${topClassName}`} id={id}>
```

Comment:
Better to use `ClassNames` to construct the className

Reply:
Thanks will switch to using ClassNames